src/h5web/providers/context.ts

```diff
@@ -4,7 +4,7 @@ import type { HDF5Id, HDF5Value, HDF5Metadata } from './models';
 export abstract class ProviderAPI {
   abstract domain: string;
   abstract async getMetadata(): Promise<HDF5Metadata>;
-  abstract async getValue(id: HDF5Id): Promise<HDF5Value>;
+  abstract async getValue(id: HDF5Id): Promise<HDF5Value | undefined>;
```

Comment:
Making the API's `getValue` signature a bit more accurate. It doesn't change anything in practice.